docs/src/pages/[platform]/components/radiogroupfield/examples/BasicExample.tsx

```diff
@@ -1,13 +1,9 @@
 import { Radio, RadioGroupField } from '@aws-amplify/ui-react';
 
-const options = ['html', 'css', 'javascript'];
-
 export const BasicExample = () => (
-  <RadioGroupField label="Language" name="language2" defaultValue="html">
-    {options.map((option) => (
-      <Radio key={option} value={option}>
-        {option}
-      </Radio>
-    ))}
+  <RadioGroupField label="Language" name="language" defaultValue="HTML">
+    <Radio value="HTML">HTML</Radio>
+    <Radio value="CSS">CSS</Radio>
+    <Radio value="JavaScript">JavaScript</Radio>
```

Comment:
Fix for same-name issue
```suggestion
  <RadioGroupField label="Animal" name="animal" defaultValue="HTML">
    <Radio value="Dog">Dog</Radio>
    <Radio value="Cat">Cat</Radio>
    <Radio value="Bird">Bird</Radio>
```